packages/test/test-utils/package.json

```diff
@@ -133,6 +132,7 @@
 		"@fluidframework/routerlicious-driver": "workspace:~",
 		"@fluidframework/runtime-definitions": "workspace:~",
 		"@fluidframework/runtime-utils": "workspace:~",
+		"@fluidframework/synthesize": "workspace:~",
```

Comment:
Unfortunately, this is needed to accommodate the types copied from `aqueduct`. Still, this package is lower in the dependency graph (and was previously depended upon transitively), so I consider the trade an improvement. Since the types added here are internal only, we can make breaking changes to them as appropriate in the future.